docs/content/error/$compile/missingattr.ngdoc

```diff
@@ -0,0 +1,6 @@
+@ngdoc error
+@name $compile:missingattr
+@fullName Missing required attribute
+@description
+
+This error may occur only when `$compileProvider.strictComponentBindingsEnabled` is set to `true`.han all attributes without `?` must be set. If one or more aren't set than first one will throw an error.
```

Comment:
There are some typos in this sentence. Can you also limit lines to 100 chars max.